starwars/src/App.js

```diff
@@ -27,6 +29,53 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        {this.state.starwarsChars.map((starwar) => {
+          return (
+            <div className="starWars">
```

Comment:
You'll want to put a 'unique' Key on this Div so that as it iterates, React knows how to keep track of it. I bet you have a couple of warning statements in your console :) This would be how to solve that!